src/02-data-types.js

```diff
@@ -36,7 +40,9 @@ function convertToNumber(val) {}
  *  convertToString(99);
  *  //> "99"
  */
-function convertToString(val) {}
+function convertToString(val) {
+  return val.toString();
```

Comment:
`toString()` definitely works in the case but remember we can also simply write : `val +""` to convert our val implicitly to a string